### ISSUE TITLE
Order of i3ipc_connection_get_workspaces replies

### ISSUE THREAD

**Issue by cornerman** [acrisci/i3ipc-glib#18]
Just found this discrepency between the `i3-msg` cli and `i3ipc-glib` and was curious why. The sort order of the workspace replies is reversed in the latter.

I guess we are `prepending` to the reply list [here](https://github.com/acrisci/i3ipc-glib/blob/57317a6bb105e0085cc786ca4cb93bbfc5726269/i3ipc-glib/i3ipc-connection.c#L1117) because of performance consideration. Should we maybe reverse the list afterwards to keep the correct order of replies? Maybe we do this in other places as well?

**Comment by acrisci** [comment]
Is this still an issue?

**Comment by cornerman** [comment]
Yep, this is still the case and I would consider it a bug - though not high priority.
